### PR TITLE
Upgrade to Cosmos SDK v0.44.8

### PR DESCRIPTION
**Pull request by nnkken**
Note that the `unsafe-reset-all` command is moved to under `tendermint` subcommand.

I found that `test-chains/prefix-test/completion.sh` is using this command, but I don't know how to update it, may @chihimng help?

cc @rickmak

### CODE REVIEW

**Comment by chihimng** [comment]
Ref issue #190

File: go.mod [hunk 1]
```diff
@@ -95,45 +95,43 @@ require (
...
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1 // indirect
-	golang.org/x/text v0.3.6 // indirect
+	golang.org/x/text v0.3.7 // indirect
 	google.golang.org/protobuf v1.27.1 // indirect
-	gopkg.in/ini.v1 v1.63.2 // indirect
+	gopkg.in/ini.v1 v1.66.2 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 	nhooyr.io/websocket v1.8.6 // indirect
 )
```
Comment (chihimng) [inline go.mod:130]:
```suggestion
// point sdk to fork and follow replaces at https://github.com/cosmos/cosmos-sdk/blob/v0.44.8/go.mod
```

Add notes on how we got these entries?

**Review by chihimng — CHANGES_REQUESTED** [review]
Otherwise LGTM 🙏

**Review by chihimng — APPROVED** [review]

**Comment by chihimng** [comment]
Merge first, follow up the script later

**Comment by nnkken** [comment]
Oh just found that that's something I generated locally when running tests... 😂

**Pull request merged**